dataplane/dplanerc/interface.go

```diff
@@ -983,7 +923,8 @@ func (ni *Reconciler) setupPorts(ctx context.Context) error {
 		data := &interfaceData{}
 
 		portResp, err := ni.portClient.CreatePort(ctx, &saipb.CreatePortRequest{
-			Switch: ni.switchID,
+			Switch:     ni.switchID,
+			HwLaneList: []uint32{uint32(idx)},
```

Comment:
Does this assume only one lane for each port?

Reply:
yes